src/components/FilePreviewer.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import FileIcon from './FileIcon';
+import LoadingIndicator from './LoadingIndicator';
+import type { FileUpload } from '../types';
+
+type Props = {
+  uploads?: FileUpload[];
+  handleRemove?: (id: string) => any;
+  handleRetry?: (id: string) => any;
+  handleFiles?: (files: File[]) => any;
```

Comment:
why any? should use `void` if they don't have a return value or `unknown` if they do return something but not sure what it can be

Reply:
We don't really use the return type of any of these callbacks. They could be `void`, but I wasn't sure if that made sense since technically any app using `react-file-utils` might already be passing down callbacks that do return something, even if it's not actually used.

Probably better to turn it into `unknown` instead, so will do that.